shell/browser/api/electron_api_web_contents.cc

```diff
@@ -2889,7 +2889,7 @@ v8::Local<v8::Promise> WebContents::TakeHeapSnapshot(
       electron_renderer.get());
   auto* raw_ptr = electron_renderer.get();
   (*raw_ptr)->TakeHeapSnapshot(
-      mojo::WrapPlatformFile(file.TakePlatformFile()),
+      mojo::WrapPlatformFile(base::ScopedPlatformFile(file.TakePlatformFile())),
```

Comment:
`TakePlatformFile` returns a `ScopedPlatformFile`, explicit initialization shouldn't be needed.

Reply:
it doesn't unfortunately: https://source.chromium.org/chromium/chromium/src/+/master:base/files/file.h;l=196?q=TakePlatformFile&ss=chromium - this fails otherwise

Reply:
I must looked at the the wrong File class, thanks for clarifying.